test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -243,7 +244,8 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 		} );
 	} );
 
-	describe( 'Basic Public Post @canary @ie11canary @parallel', function() {
+	// @TODO: add `@ie11canary` back here once FSE plugin works with IE11 again.
```

Comment:
Can we add this back now that #40435 is merged? (Might need a rebase first.)

Reply:
Not familiar with the FSE workflow nowadays, but I think we need D40802-code to be deployed first.

Reply:
It'll need FSE plugin deploy to simple sites too — once it's done we can enable but it can be in separate PR.

Reply:
Prepped a PR to enable https://github.com/Automattic/wp-calypso/pull/40443 — good to go once D40826-code is deployed.